fastscanner/adapters/rest/scanner.py

```diff
@@ -118,3 +134,44 @@ async def websocket_realtime_scanner(
         if scanner_id:
             await service.unsubscribe_realtime(scanner_id)
             logger.info(f"Unsubscribed scanner {scanner_id}")
+
+
+@router.post("/scan")
```

Comment:
```suggestion
@router.post("")
```
"/scanners" is a restful way to call the endpoint.